server/data/data-jmap-cassandra/src/main/java/org/apache/james/jmap/cassandra/filtering/RuleDTO.java

```diff
@@ -47,13 +47,33 @@
     }
 
     public static RuleDTO from(Rule rule) {
-        return new RuleDTO(rule.getId());
+        return new RuleDTO(rule.getId().asString(),
+                rule.getName(),
+                rule.getCondition().getField().asString(),
+                rule.getCondition().getComparator().asString(),
+                rule.getCondition().getValue(),
+                rule.getAction().getAppendInMailboxes().getMailboxIds());
```

Comment:
you don't modify also the DTO?

Reply:
The DTI is *flat* so far. Should I give it a new shape to match the Rule object structure?

Reply:
Seems a good idea. Any reason to have it flat else?

Reply:
'cause I'm lazy?

Reply:
Done